parsl/tests/test_mpi_apps/test_mpi_scheduler.py

```diff
@@ -161,3 +161,28 @@ def test_MPISched_contention():
     assert task_on_worker_side['task_id'] == 2
     _, _, _, resource_spec = unpack_res_spec_apply_message(task_on_worker_side['buffer'])
     assert len(resource_spec['MPI_NODELIST'].split(',')) == 8
+
+
+@pytest.mark.local
+def test_hashable_backlog_queue():
```

Comment:
if this is a regression test, I would expect it to fail, but it doesn't with master 07a4efb14e4f0863a1b28f01a5d163e06f30c131 on my laptop

Reply:
I was surprised. It definitely fails for me! Happy we got this sorted :)